### PR TITLE
align iptables args with non-cni

### PR DESCRIPTION
**Pull request by lambdai**
InboundPorts annotation set to star only if user doesn't provide label
Use user original value otherwise.

Signed-off-by: Yuchen Dai <silentdai@gmail.com>

Fix up according to #284

### CODE REVIEW

File: cmd/istio-cni/main.go [hunk 1]
```diff
@@ -224,7 +224,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 				}
 				if !excludePod {
 					log.Infof("setting up redirect")
-					if redirect, redirErr := NewRedirect(ports, annotations); redirErr != nil {
```
Comment (rlenglet) [inline cmd/istio-cni/main.go:227]: `ports` is now unused.
Just remove the `ports` variable and modify `getKubePodInfo` to not return the ports. It should make `getK8sPodInfo` quite shorter.

File: cmd/istio-cni/redirect.go [hunk 2]
```diff
@@ -202,10 +202,9 @@ func NewRedirect(ports []string, annotations map[string]string) (*Redirect, erro
 		log.Errorf("Annotation value error for redirect ports, using ContainerPorts=\"%s\": %v",
 			redir.includePorts, valErr)
 		return nil, valErr
-	} else if !isFound || redir.includePorts == "*" {
+	} else if !isFound {
```
Comment (rlenglet) [inline cmd/istio-cni/redirect.go:205]:
```suggestion
	}
	if !isFound {
```

File: cmd/istio-cni/main.go (hunk 1 above)
Comment (lambdai) [reply cmd/istio-cni/main.go:227]: Great catch!

File: cmd/istio-cni/redirect.go (hunk 2 above)
Comment (lambdai) [reply cmd/istio-cni/redirect.go:205]: Thanks!

**Pull request merged**